Service/appsettings.Development.json

```diff
@@ -7,6 +7,7 @@
     }
   },
   "ConnectionStrings": {
-    "mysql": "Server=nephele-hw0-db.ckeyz18go9z6.us-east-1.rds.amazonaws.com; database=Project; UID=admin; password=Data123!#"
+    //"mysql": "Server=nephele-hw0-db.ckeyz18go9z6.us-east-1.rds.amazonaws.com; database=Project; UID=admin; password=Data123!#"
+    "mysql": "Server=localhost; port=3306; database=AseSharpDB; user=root; password=asesharp"
```

Comment:
Changed to a local database, which I think is what IA wants.

Reply:
Yes agreed, nice!